projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.tsx

```diff
@@ -0,0 +1,201 @@
+import { imagePath } from 'constants/urls';
```

Comment:
I converted this file to TypeScript because we seem to be moving in that direction